src/AutomatedCar/Visualization/PositionComputeObject.cs

```diff
@@ -20,7 +20,7 @@ public Point[] getScreenSquare(AutomatedCar.Models.WorldObject worldObject)
         }
         public Point getPositionFromCenter(AutomatedCar.Models.WorldObject worldObject)
         {
-            throw new NotImplementedException();
+            return new Point(this.centerCar.X - worldObject.X, this.centerCar.Y - worldObject.Y);
```

Comment:
Ez nem  worldObject.X-this.centerCar.X inkább?
A világ origója a bal felső sarokban van x tengely jobbra, y lefele pozitív, és egy olyan koordináta rendszerbe kell átszámolni, aminek az origója középpen van, x jobbra, és y felfelé a pozitív iránnyal.

Reply:
Oks, javítottam.